include/library/spdm_secured_message_lib.h

```diff
@@ -207,12 +207,25 @@ typedef uint8_t (*libspdm_secured_message_get_sequence_number_func)(
  **/
 typedef uint32_t (*libspdm_secured_message_get_max_random_number_count_func)(void);
 
-#define SPDM_SECURED_MESSAGE_CALLBACKS_VERSION 1
+/**
+ * This function translates the negotiated secured_message_version to a DSP0277 version.
+ *
+ * @param  secured_message_version  The version specified in binding specification and
+ *                                  negotiated in KEY_EXCHANGE/KEY_EXCHANGE_RSP.
+ *
+ * @return The DSP0277 version specified in binding specification,
+ *         which is bound to secured_message_version.
+ */
+typedef spdm_version_number_t (*libspdm_secured_message_get_secured_spdm_version)(
```

Comment:
These should be defined in `spdm.h` or renamed to `libspdm_`.

Reply:
I am confused.
Which should be defined in spdm.h? or renamed to libspdm\_?